azure-pipelines.yml

```diff
@@ -92,6 +92,13 @@ jobs:
           MB_ML_VER: "2014"
           NP_BUILD_DEP: "numpy==1.19.3"
           NIGHTLY_BUILD: "true"
+        py_3.10_64:
+          MB_PYTHON_VERSION: "3.10"
+          MB_ML_VER: "2014"
+          NP_BUILD_DEP: "numpy==1.21.2"
+          CYTHON_BUILD_DEP: "cython==0.29.24"
+          DOCKER_TEST_IMAGE: "multibuild/focal_{PLAT}"
```

Comment:
if Xenial is EOL, do the other envs need updating at some point?

Reply:
Yup, we actually have trusty(ubuntu 14.04) in the x86_64/i686 builds. All of those will need updating(not in this PR).

Reply:
> not in this PR

sure